docs/developers/release-checklist.md

```diff
@@ -1,16 +1,21 @@
 ## `ddev` Release Checklist 
 
-- [ ] [Create a release](https://github.com/drud/ddev/releases/new) using the github web UI, initiating a CircleCI tag build
+- [ ] For each container directory in `containers/`, `make push VERSION=vXXX` to ensure the current container version exists for building and testing
+- [ ] Update the default container versions in `pkg/version/version.go` and create a pull request
+- [ ] Once all updates have been merged into the master branch, create a tag for the release version, initiating a tag build
```

Comment:
Creating the release creates the tag, not sure why github refers to them as different things. So this step really isn't needed I don't think

Reply:
Ah, I had incorrectly assumed that you'd be able to create a tag independently of a release on the Github UI.  I'll roll these steps together again.